homeassistant/components/climate/generic_thermostat.py

```diff
@@ -268,7 +267,8 @@ def min_temp(self):
         if self._min_temp:
             return self._min_temp
 
-        return DEFAULT_MIN_TEMP
+        # get default temp from super class
+        return ClimateDevice.min_temp.fget(self)
```

Comment:
Why can't you call `super().min_temp` like you do in the other platforms?

Reply:
I was just reverting to the previous version... but I agree with your comment, will do the change